org.metaborg.spoofax.core/src/main/java/org/metaborg/spoofax/core/stratego/IStrategoCommon.java

```diff
@@ -67,6 +91,8 @@
      *            Input term
      * @param strategy
      *            Name of the strategy to call.
+     * @param termArguments
+     *            Term arguments the strategy gets called with
```

Comment:
This `@param` looks to be in the wrong javadoc, the one above has the `termArguments` argument

Reply:
Done